packages/smooth_app/lib/data_models/continuous_scan_model.dart

```diff
@@ -32,7 +32,8 @@ class ContinuousScanModel with ChangeNotifier {
   late DaoProduct _daoProduct;
   late DaoProductList _daoProductList;
 
-  bool get hasMoreThanOneProduct => getBarcodes().length > 1;
+  int get productsCount => getBarcodes().length;
```

Comment:
`getBarcode()` is public and already used outside this class in `ButtonsRow`.
I think we should get rid of both the previous `hasMoreThanOneProduct` method and your current `productsCount`. Their added value is low and it adds confusion when reading the code: "why the hell did the developer add this method?"